packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -91,26 +134,18 @@ export class ErrorBars
 
     cap: ErrorBarCapConfig = new ErrorBarCapConfig();
 
-    private readonly cartesianSeries: AnyCartesianSeries;
+    private readonly cartesianSeries: ErrorBoundCartesianSeries;
     private readonly groupNode: _Scene.Group;
     private readonly selection: _Scene.Selection<ErrorBarNode>;
     private nodeData: (ErrorBarPoints | undefined)[] = [];
 
     private dataModel?: AnyDataModel;
     private processedData?: AnyProcessedData;
 
-    constructor(ctx: _ModuleSupport.SeriesContext) {
+    constructor(ctx: SeriesContext) {
         super();
 
-        const supportedSeriesTypes = ['bar', 'line', 'scatter'];
-        if (!supportedSeriesTypes.includes(ctx.series.type)) {
-            throw new Error(
-                `AG Charts - unsupported series type '${
-                    ctx.series.type
-                }', error bars supported series types: ${supportedSeriesTypes.join(', ')}`
-            );
-        }
```

Comment:
Regarding my previous comment, this looks good enough - perhaps we could just use the options typing for the `supportedSeriesTypes` array to get some compiler assistance that the elements are valid?